runhouse/resources/hardware/cluster.py

```diff
@@ -177,6 +172,38 @@ def save_config_to_cluster(
             node=node or "all",
         )
 
+    def update_cluster_config_on_cluster(
+        self, items_to_update: Union[dict, list[dict]]
+    ):
+        """
+        Update specific items on the cluster config saved on the cluster.
+        :param items_to_update: The item(s) to be updated in the config. If only one item should be updated, pass
+        it as a dictionary. e.g: {"den_status_ping_interval": -1}. If fre items need to be updated, pass them as list
+        of dictionaries, e.g: [{"den_status_ping_interval": -1}, {log_length_in_den: 30}]
+        """
+        current_config_on_cluster = self.run([f"cat {CLUSTER_CONFIG_PATH}"])
```

Comment:
what's the benefit of running the command on the cluster, as opposed to just calling `self.config()`?